automation/test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -ex
+
+kubectl() { cluster/kubectl.sh "$@"; }
+
+export CLUSTER_PROVIDER=$TARGET
+
+# Make sure that the VM is properly shut down on exit
+trap '{ make cluster-down; }' EXIT SIGINT SIGTERM SIGSTOP
+
+make cluster-down
+make cluster-up
+make cluster-sync
+
+DEPLOY_DIR=_out/ VERSION=latest make generate-manifests
```

Comment:
I think we don't need this the `make cluster-sync` already do it.

take a look on the logs here https://kubevirt-master-stdci-production.apps.ovirt.org/job/kubevirt_cluster-network-addons-operator_standard-check-pr/61//artifact/check-patch.k8s-1.11.0.el7.x86_64/mock_logs/script/stdout_stderr.log

you will the the manifest tool run two times.

Reply:
The problem is that manifests that are already generated use future version (so now, manifests are under 0.5.0, once we bump version it will change). I wanted to follow an explicit way so we know where to look for manifests. However, there is another problem, we overwrite the package file. That might be a surprise if we use it one day. Lemme change it a bit.